sigar_linux.go

```diff
@@ -562,7 +562,29 @@ func (self *DiskList) Get() error {
 }
 
 func (self *ProcessList) Get() error {
-	return notImplemented()
+	pids := ProcList{}
+	err := pids.Get()
+	if err != nil {
+		return err
+	}
+
+	processes := make([]Process, 0, len(pids.List))
+	for _, pid := range pids.List {
+		var process Process
+
+		// Gather each composed struct, ignoring any errors.
```

Comment:
Is there anything useful you can do with the errors?  I guess logging would also be problematic since the same error would likely come up every call.